[PATHS]
DATA_FILE_PATH = ./tests/resources/

[PDF URIs]
PDF_URI = http://arxiv.org/pdf/cs/9308101v1
NOT_FOUND_PDF_URI = http://arxiv.org/pdf/cs/cvzqcdcsdcdsfcdfvdfvdfvfdvdv
NOT_PDF_URI = https://docs.python.org/3/tutorial/errors.html
WRONG_PDF_URI = ?!

[REFERENCES]
PDF_DATA_REFERENCE_FILE_NAME = pdf_reference.pdf
PDF_CONTENT_REFERENCE_FILE_NAME = pdf_content_reference.txt
PDF_METADATA_REFERENCE_FILE_NAME = pdf_metadata_reference.json
FEED_DATA_REFERENCE_FILE_NAME = feed_reference.txt
PDF_METADATAS_REFERENCE_FILE_NAME = pdf_metadatas_reference.json
CLEAN_REFERENCES_REFERENCE_FILE_NAME = clean_references_reference.json
REFEXTRACT_REFERENCES_REFERENCE_FILE_NAME = refextract_references_reference.json

[ArXivParser]
time_step = 604800
arxiv_url = https://export.arxiv.org/api/
cat = cs.ai
max_results = 1000

[Cooldown Manager]
cooldown_manager_uri = http://172.17.0.2:5000/

[tool:pytest]
testpaths = tests

[coverage:run]
branch = True
source = .